plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -300,6 +303,8 @@ public ExecutionResult createFileInVR(final String routerIp, final String path,
         final File permKey = new File("/root/.ssh/id_rsa.cloud");
         String error = null;
 
+        s_logger.debug("Creating file in VR " + filename);
```

Comment:
Please wrap in a ``if (s_logger.isDebugEnabled())`` block to avoid unnecessary string creation due to pre-method string concatenation when ``DEBUG`` is not enabled.